skuba-update/tox.ini

```diff
@@ -1,71 +1,32 @@
 [tox]
+minversion = 2.5.0
 skip_missing_interpreters = True
-skipsdist = True
-envlist =
-    check,
-    py36,
-    py34,
+envlist = check, py38, py36, py34
+skipdist = True
```

Comment:
This seems wrong, the old version `skipsdist` looks right https://tox.readthedocs.io/en/latest/config.html#conf-skipsdist